pinot-controller/src/test/java/org/apache/pinot/controller/helix/PinotControllerModeTest.java

```diff
@@ -41,8 +46,7 @@ public void setUp() {
   }
 
   @Test
-  public void testHelixOnlyController()
-      throws Exception {
+  public void testHelixOnlyController() throws Exception {
```

Comment:
Please update IDE to use Pinot Style, and reformat all the changes

Reply:
Reformatted.